web/src/lib/components/asset-viewer/asset-viewer-nav-bar.svelte

```diff
@@ -182,7 +182,12 @@
       {#if !asset.isReadOnly || !asset.isExternal}
         <CircleIconButton isOpacity={true} icon={mdiDeleteOutline} on:click={() => dispatch('delete')} title="Delete" />
       {/if}
-      <div use:clickOutside on:outclick={() => (isShowAssetOptions = false)}>
+      <div
+        use:clickOutside={{
```

Comment:
I didn't know you could do this! Amazing!

Reply:
It's from the cool work done in #7677!